aml-development/frontend
# fix(storefront): refactor storefront and improve performance

Removes the need for "dummy" wrapper component in storefront by separating each section into its own component.

Each section now loads independently so the user doesn't have to wait for the entire storefront before receiving data.

**How to Test**
Pull this branch
Make sure storefront loads appropriately for all different users
Make sure the See More pages also load the appropriate data
If there are serious performance issues when it comes to storefront interaction, make note of it in this PR
